### DISCUSSION TITLE
Databases: Adding mongodb course

### DISCUSSION THREAD

**Discussion by Upendra2003** [TheOdinProject/curriculum#26638] [Ideas]
### Describe your suggestion

In the databases course, we currently only offer instruction on MySQL. Considering the widespread adoption of MongoDB in today's industry, it may be beneficial to consider adding a MongoDB course to the curriculum, as many companies now rely on this database system.

### Path

Other / NA

### Lesson Url

https://www.mongodb.com/docs/

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### (Optional) Additional Comments

I thoroughly enjoyed this project and I'm eager to contribute further.

**Comment by duckvshuman** [comment]
They do teach mongodb if you follow the full-stack JavaScript path.

**Comment by CouchofTomato** [comment]
I've converted this to a discussion because it is better served here.

**Comment by alexwittwer** [comment]
It probably varies on your location and available jobs.

For me, locally, theres a lot of older institutions, government contractors, banks and financial places that use MySQL. Very few startups here so not much MongoDB.

Which I get that mongoDB is nice and shiny, but completely useless unless I'm willing to completely uproot, or to just learn SQL.
